Shared/Samples/Identify features in WMS layer/IdentifyFeaturesInWMSLayerView.swift

```diff
@@ -0,0 +1,112 @@
+
+    /// The tapped screen point.
+    @State var tapScreenPoint: CGPoint?
+    
+    /// The string text for the identify layer results overlay.
+    @State var overlayText = "Tap on the map to identify features in the WMS layer."
+
+    /// The error shown in the error alert.
+    @State private var error: Error?
+
+    var body: some View {
+        MapViewReader { mapViewProxy in
+            MapView(map: map)
+                .onSingleTapGesture { screenPoint, _ in
+                    tapScreenPoint = screenPoint
+                }
+                .task {
+                    do {
+                        // Create a WMS layer from a URL and load it.
+                        let wmsLayer = WMSLayer(url: .EPAWaterInfo, layerNames: ["4"])
+                        try await wmsLayer.load()
+                        
+                        // Add the layer to the map.
+                        map.addOperationalLayer(wmsLayer)
+                        
+                        waterInfoLayer = wmsLayer
+                    } catch {
+                        self.error = error
+                    }
+                }
+                .task(id: tapScreenPoint) {
+                    // Identify on WMS layer using the screen point.
+                    webViewText = if let screenPoint = tapScreenPoint,
+                                     let waterInfoLayer,
+                                     // Identify feature on water info layer
+                                     let identifyResult = try? await mapViewProxy.identify(
+                                        on: waterInfoLayer,
+                                        screenPoint: screenPoint,
+                                        tolerance: 2
+                                     ),
+                                     // Convert the result to text.
+                                     let feature = identifyResult.geoElements.first,
+                                     let htmlText = feature.attributes["HTML"] as? String,
+                                     // Display the HTML table if it has an OBJECTID column.
+                                     htmlText.contains("OBJECTID") {
+                        htmlText
+                    } else {
+                        String()
+                    }
+                }
+                .overlay(alignment: .top) {
+                    VStack {
+                        Text(overlayText)
+                            .frame(maxWidth: .infinity, alignment: .center)
+                            .padding(.top)
+                        ScrollView(.horizontal) {
+                            WebView(htmlString: webViewText)
+                                // Set the width so the html is readable.
+                                .frame(width: 800, height: webViewText.isEmpty ? 0 : 95)
+                        }
+                        .padding(.leading)
+                        .padding(.trailing)
```

Comment:
```suggestion
                        .padding(.horizontal)
```

Not a big deal, just a little simpler.